docs/getting-started/index.mdx

```diff
@@ -6,174 +6,377 @@ pagination_next: http/index
-Version                       3.0.0
-Region                        United States (us)
-Latency                       78ms
+Account                       Foo Bar (Plan: All)
+Version                       3.18.4
+Region                        United States (California) (us-cal-1)
+Latency                       37ms
 Web Interface                 http://127.0.0.1:4040
-Forwarding                    https://84c5df474.ngrok-free.dev -> http://localhost:8080
+Forwarding                    https://<GENERATED_NGROK_DOMAIN> -> http://localhost:8080
 
 Connections                   ttl     opn     rt1     rt5     p50     p90
                               0       0       0.00    0.00    0.00    0.00
 ```
 
-Open the Forwarding URL in your browser and you will see your web application.
+Open the **Forwarding** URL in your browser and you will see your service.
 
 - This URL is available to anyone on the internet. Test it out by sending it to
   a friend!
-- Your app is available over HTTPS (notice the 🔒 in your browser window) with
+- We call this URL an **agent endpoint**.
+- Your service is available over HTTPS (notice the 🔒 in your browser window) with
   a valid certificate that ngrok automatically manages for you.
 
-## Step 4: Always use the same domain
+## Step 3: Always use the same domain
 
-If you want to keep the same URL each time you use ngrok, [create a static
-domain on your dashboard](https://dashboard.ngrok.com/cloud-edge/domains) and
-then use the `--url` flag to ask the ngrok agent to use it. First, stop
-ngrok with `Ctrl+C` and then run ngrok again:
+If you want to keep the same URL each time you use the ngrok agent, [create a
+static domain on your dashboard](https://dashboard.ngrok.com/domains/new)
+and then use the `--url` flag to ask the ngrok agent to use it. First, stop
+ngrok with `Ctrl+C` and then run ngrok again with your new domain:
 
+```bash
+ngrok http 8080 --url <YOUR_NGROK_DOMAIN>
 ```
-ngrok http 8080 --url jumpy-red-mollusk.ngrok-free.app
-```
 
-## Step 5: Secure your app
+## Step 4: Write your first Traffic Policy rules
+
+You may not want anyone to be able to access your app or API on your even though
+you've made it publicly available. With ngrok, you can add authentication and
+other traffic management rules, without making any changes to your service,
+using our [Traffic Policy](/docs/http/traffic-policy/index.mdx) engine.
+
+Traffic Policy is a powerful configuration language for your endpoints, letting
+you flexibly filter, match, manage, and orchestrate traffic exactly as you need.
 
-You may not want everyone to be able to access your application. ngrok can
-quickly add authentication to your app without any changes. If your Google
-account is `alan@example.com`, you can restrict access only for yourself by
-running ngrok with:
+Let's look at a basic Traffic Policy rule to illustate the key concepts:
 
+```yaml
+on_http_request:
+  - expressions:
+      - ""conn.ts.start < timestamp('2999-12-31T23:59:59Z')""
```

Comment:
this expression doesn't make much sense. can we use something someone might actually use here as an example?